src/Altinn.Notifications/appsettings.json

```
@@ -1,8 +1,8 @@
 {
   "PlatformSettings": {
     "ApiProfileEndpoint": "http://localhost:5030/profile/api/v1/",
-    "ApiRegisterEndpoint": "http://localhost:5020/register/api/v1/"
-
+    "ApiRegisterEndpoint": "http://localhost:5020/register/api/v1/",
+    "ApiAuthorizationEndpoint": "http://localhost:5030/authorization/api/v1/"
```

I though authorization was port 5050. Will you double check. Shouldn't be the same as profile

```suggestion
    "ApiAuthorizationEndpoint": "http://localhost:5050/authorization/api/v1/"
```

---

Is the Dockerfile in the authorization project wrong then? How does that work? Is there an override somewhere?
`EXPOSE 5030`

---

Probably. We set the[ port in the helm chart](https://dev.azure.com/brreg/_git/altinn-studio-ops?path=/deploy/altinn-platform/altinn-authorization/values.yaml&version=GBmaster&line=7&lineEnd=8&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents) and there 5050 is used for authorization.

---

Okay, but the helm chart is not involved when I run the application in docker/podman.

---

Issue has been fixed in authz repo, so we can update here as well.